ui/src/store/modules/projects.ts

```diff
@@ -66,14 +66,6 @@ const actions: ActionTree<ProjectsState, RootState> = {
       context.commit('removeOne', project)
     })
   },
-
-  async loadSources (context, project) {
-    await handleAPIError(context, async () => {
-      const sources = await client.projects.getSources(project)
-
-      context.commit('storeSources', { project, sources })
-    })
-  },
```

Comment:
Where did this go?

Reply:
It went in `sources/loadForProject`. It's a remain of a previous refactoring.

Reply:
It went in sources/loadForProject. It's a remain of a previous refactoring.